drivers/sensor/lis2dw12/lis2dw12.c

```diff
@@ -319,6 +319,20 @@ static int lis2dw12_init(const struct device *dev)
 	}
 #endif /* CONFIG_LIS2DW12_TRIGGER */
 
+	LOG_INF("high pass reference mode is %d", (int)cfg->hp_ref_mode);
```

Comment:
Similar comments in this patch -- LOG_INF seems a bit noisy for these, no?

Reply:
Changed them to LOG_DBG. Totally makes sense!